Makefile

```diff
@@ -65,6 +63,12 @@ mypy-all:
 	# Remaining errors in tests:
 	mypy --config-file /dev/null raiden --ignore-missing-imports | grep error | wc -l
 
+linting-tools:
```

Comment:
maybe use `mypy` here instead of providing the calls in the recipe? otherwise LGTM